.env

```diff
@@ -0,0 +1 @@
+REACT_APP_YT_API_KEY=AIzaSyBWg9DbhIDfhVIODXXx4bK9-tfXkSldPPw
```

Comment:
Never commit keys or credentials in a code repository. Look for a way to remove these completely from your git history and find out how to provide them within your hosting platform, instead.